src/Http/Http.Abstractions/src/PathString.cs

```diff
@@ -20,14 +20,14 @@ namespace Microsoft.AspNetCore.Http
         /// </summary>
         public static readonly PathString Empty = new PathString(string.Empty);
 
-        private readonly string? _value;
+        private readonly string _value;
```

Comment:
⚠️ This change doesn't handle `default(PathString)` as well as the original code. I'd use the new C# 9 `MemberNotNullWhenAttribute` instead.